Testscripts/Linux/dpdkUtils.sh

```diff
@@ -318,8 +324,11 @@ function Install_Dpdk () {
 	LogMsg "MLX_PMD flag enabling on ${1}"
 	if type Dpdk_Configure > /dev/null; then
 		echo "Calling testcase provided Dpdk_Configure(1) on ${1}"
-		ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && meson build"
-		#ssh ${1} "sed -ri 's,(MLX._PMD=)n,\1y,' ${LIS_HOME}/${DPDK_DIR}/build/.config"
+		if [[ ${DISTRO_NAME} == rhel || ${DISTRO_NAME} == centos ]]; then
+			ssh ${1} "cd ${LIS_HOME}/${DPDK_DIR} && PATH=$PATH:/opt/rh/rh-python36/root/usr/bin/ && meson build"
```

Comment:
If we are moving forward to python38 or later. this path is not valid

Reply:
For DPDK installation meson version 0.47.1 is [required ](http://doc.dpdk.org/guides/prog_guide/build-sdk-meson.html)which comes with python36 minimum in RHEL. That's why I included it.